<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmVessel</class>
 <widget class="QWidget" name="frmVessel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QLabel" name="lbHeader">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(247, 247, 247)</string>
     </property>
     <property name="text">
      <string>WRITE SOMETHING HERE LATER &lt;---</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Select sampled vessel:</string>
     </property>
     <property name="buddy">
      <cstring>tableView</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushNew">
         <property name="toolTip">
          <string>Create new record</string>
         </property>
         <property name="statusTip">
          <string>Create new record</string>
         </property>
         <property name="whatsThis">
          <string>Creates new record</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="app_new.qrc">
           <normaloff>:/app_new/filenew.png</normaloff>:/app_new/filenew.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="toolTip">
          <string>Add vessel not present on this list</string>
         </property>
         <property name="statusTip">
          <string>Add vessel not present on this list</string>
         </property>
         <property name="whatsThis">
          <string>This tool allows to setup temporary frame changes concerning adding new vessels or importing them from another port</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupDetails">
     <property name="title">
      <string>Details</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Vessel:</string>
              </property>
              <property name="buddy">
               <cstring>cmbVessel</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cmbVessel"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Sample Origin:</string>
              </property>
              <property name="buddy">
               <cstring>cmbOrigin</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cmbOrigin"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Status:</string>
              </property>
              <property name="buddy">
               <cstring>cmbStatus</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cmbStatus"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupFT">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QLabel" name="label_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Estimated</string>
             </property>
             <property name="buddy">
              <cstring>spinET</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Calculated</string>
             </property>
             <property name="buddy">
              <cstring>spinCT</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Number of fishing trips:</string>
             </property>
             <property name="buddy">
              <cstring>spinET</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spinET"/>
           </item>
           <item row="1" column="2">
            <widget class="QSpinBox" name="spinCT"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Comments:</string>
          </property>
          <property name="buddy">
           <cstring>textComments</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="textComments">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushPrevious">
       <property name="text">
        <string>Previous</string>
       </property>
       <property name="icon">
        <iconset resource="app_new.qrc">
         <normaloff>:/app_new/back.png</normaloff>:/app_new/back.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushNext">
       <property name="text">
        <string>Next</string>
       </property>
       <property name="icon">
        <iconset resource="app_new.qrc">
         <normaloff>:/app_new/forward.png</normaloff>:/app_new/forward.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tableView</tabstop>
  <tabstop>pushNew</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>cmbVessel</tabstop>
  <tabstop>cmbOrigin</tabstop>
  <tabstop>cmbStatus</tabstop>
  <tabstop>textComments</tabstop>
  <tabstop>spinET</tabstop>
  <tabstop>spinCT</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>pushPrevious</tabstop>
  <tabstop>pushNext</tabstop>
 </tabstops>
 <resources>
  <include location="app_new.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushNew</sender>
   <signal>clicked()</signal>
   <receiver>frmVessel</receiver>
   <slot>createRecord()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>676</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>createRecord()</slot>
 </slots>
</ui>
